src/hotspot/cpu/riscv/globals_riscv.hpp

```diff
@@ -103,6 +103,7 @@ define_pd_global(intx, InlineSmallCode,          1000);
   product(bool, UseZba, false, EXPERIMENTAL, "Use Zba instructions")             \
   product(bool, UseZbb, false, EXPERIMENTAL, "Use Zbb instructions")             \
   product(bool, UseZbs, false, EXPERIMENTAL, "Use Zbs instructions")             \
+  product(bool, UseZfh, false, EXPERIMENTAL, "Use Zfh instructions")             \
```

Comment:
`fcvt.s.h`/`fcvt.h.s`/`fmv_h_x`/`fmv.x.h` also belongs to Zfhmin[1], we can use `UseZfhmin` instead of `UseZfh` here.

1. https://github.com/riscv/riscv-isa-manual/blob/cb3b9d1dcdacefbde6602ada7a0050f5c723ddee/src/zfh.tex#L377-L391

Reply:
Make sense. Fixed.